api/src/controllers/appMilestone.js

```diff
@@ -53,37 +53,73 @@ router.post(
 
     // USER SURVEY:
     if (req.headers.appversion < 205) return res.status(200).send({ ok: true });
-    if (req.headers.appversion >= 238 && user.createdAt && dayjs(user.createdAt).isBefore(dayjs().subtract(90, "day"))) {
-      const superUserFeature = await prisma.appMilestone.findUnique({
-        where: { id: `${user.id}_@SuperUserFeature` },
+    if (req.headers.appversion >= 239 && user.createdAt && dayjs(user.createdAt).isBefore(dayjs().subtract(90, "day"))) {
+      const super90UserFeature = await prisma.appMilestone.findUnique({
+        where: { id: `${user.id}_@Super90UserFeature` },
       });
-      if (!superUserFeature) {
+      if (!super90UserFeature) {
         await prisma.appMilestone.create({
           data: {
-            id: `${user.id}_@SuperUserFeature`,
+            id: `${user.id}_@Super90UserFeature`,
             userId: user.id,
             date: dayjs().format("YYYY-MM-DD"),
           },
         });
         return res.status(200).send({
           ok: true,
           showInAppModal: {
-            id: "@SuperUserFeature",
+            id: "@Super90UserFeature",
             title: "Merci d'être avec nous",
             content:
               "Bravo, vous ête sur Oz depuis quelques mois et votre expérience nous est précieuse. Nous serions ravis de recueillir vos recommandations pour continuer à améliorer l'application :)",
             CTATitle: "Donner mon avis",
-            CTANavigation: ["SUPER_NPS_SCREEN"],
+            CTANavigation: ["SUPER_NPS_SCREEN", { days: "90" }],
             secondaryButtonTitle: "Plus tard",
             CTAEvent: {
-              category: "NPS",
+              category: "SUPER_90_NPS",
               action: "PRESSED_FROM_NEW_FEATURE_MODAL",
               name: "FROM_NEW_FEATURE",
             },
           },
         });
       }
     }
+    if (req.headers.appversion >= 239 && user.createdAt && dayjs(user.createdAt).isBefore(dayjs().subtract(30, "day"))) {
+      const super90UserFeature = await prisma.appMilestone.findUnique({
```

Comment:
mets un comentaire ici, genre: `// if the user skipped the 30 days, he will have got the 90 days - if he got the 90 days, we don't show him the 30 days`